### ISSUE TITLE
Support RichText conversion

### ISSUE THREAD

**Issue by rohannair** [sanity-io/contentful-to-sanity#7]
Contentful's rich text type isn't supported by this tool: https://github.com/sanity-io/contentful-to-sanity/blob/master/src/transformSchema.js#L1

Contentful's RESTful response for a Rich Text field follows this general schema:

```javascript
{
  ...
  fields: {
    ...
    body: {
      data: {},
      content: [
        {
          data: {},
          marks: [
            {
              type
            }
          ],
          value,
          nodeType
        }
      ],
      nodeType
    }
  }
}
```

Objects within content can have nested content lists as well, so it's likely that this requires a touch of iteration.

**Comment by rohannair** [comment]
This would also address #6

**Comment by WarHatch** [comment]
Greetings, I've also ran into this bug. Is there any progress on fixing it?

**Comment by rohannair** [comment]
I wouldn't classify this as a bug. It hasn't been implemented yet. The work requires writing the actual conversion algorithm.

**Comment by kmelve** [comment]
Good point (about the bug thing) @rohannair 

! Without any promises, I think we actually have something in the works here, will follow-up when the developer is back from vacation. Stay tuned.

**Comment by alex-tgk** [comment]
> Good point (about the bug thing) @rohannair!
> 
> Without any promises, I think we actually have something in the works here, will follow-up when the developer is back from vacation. Stay tuned.

Is there any progress to be made on this ticket? 
Looking to convert to sanity and having this conversion tool would be a crucial factor for having an easy transition

**Comment by perminder-klair** [comment]
Hi guys,

Any progress on this? looking forward to moving a few sites to sanity, just this issue is holding it.

thank you!

**Comment by kmelve** [comment]
Hi! We're working on it! Technically it seems to be no problem, but there are some decisions we need to take in terms of how to translate breaks to Portable Text.

**Comment by cashmagnet** [comment]
Hey! 

I'd also be really interested in knowing how this progresses - I'd really like to move a few sites over also - and this would make that possible :-) 

Thanks!

**Comment by Devin-Chaves** [comment]
I'm also really interested. We'll be moving from contentful rich text to portable text so this would be a welcome improvement.

**Comment by runeb** [comment]
We are now releasing version 1.1.4 of this tool which supports RichText, so if this is still relevant to the users in this thread I encourage you to check that out!

**Comment by runeb** [comment]
Correction: The version released is actually `2.0.0`
